bee-network/src/init.rs

```diff
@@ -0,0 +1,216 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg(feature = "full")]
+
+use super::{
+    config::NetworkConfig,
+    peer::list::{PeerList, PeerListWrapper},
+    service::{
+        command::command_channel,
+        controller::{NetworkCommandSender, NetworkEventReceiver},
+        event::{event_channel, Event, InternalEvent},
+    },
+    types::{PeerInfo, PeerRelation},
+    Keypair, PeerId,
+};
+
+use crate::{
+    alias,
+    network::host::NetworkHostConfig,
+    service::service::{self, NetworkServiceConfig},
+};
+
+use libp2p::identity;
+use log::info;
+use once_cell::sync::OnceCell;
+
+pub mod global {
+    use super::*;
+
+    static RECONNECT_INTERVAL_SECS: OnceCell<u64> = OnceCell::new();
+    static NETWORK_ID: OnceCell<u64> = OnceCell::new();
+    static MAX_UNKNOWN_PEERS: OnceCell<usize> = OnceCell::new();
+
+    pub fn set_reconnect_interval_secs(reconnect_interval_secs: u64) {
+        if cfg!(test) {
```

Comment:
```rust
RECONNECT_INTERVAL_SECS
    .set(reconnect_interval_secs)
    .err()
    .filter(|_| !cfg!(test))
    .expect_none()
```

Perhaps?

Reply:
`expect_none` on option is unstable

Reply:
also a question for you: Do you think this is very readable? It's very concise, but the double negation is a bit wacky. haha

Reply:
I agree with Alex, I find the filter particularly confusing even though I rationally know what it means

Reply:
Alright. Closing this thread, and keeping the current version. `expect_none` is unstable anyway :smiley: